galleries/plot_types/basic/scatter_plot.py

```diff
@@ -26,4 +28,4 @@
 ax.set(xlim=(0, 8), xticks=np.arange(1, 8),
        ylim=(0, 8), yticks=np.arange(1, 8))
 
-plt.show()
+plt.show()
```

Comment:
```suggestion
plt.show()

```

(flake8 complains of no new line at the end of the file